src/lib/button/button.scss

```diff
@@ -79,6 +80,10 @@
   border-radius: inherit;
   pointer-events: none;
   opacity: 0;
+
+  @include md-high-contrast {
+    background-color: rgba(white, 0.5);
```

Comment:
If we make these styles part of the theme, they can simply use the theme's background and foreground palettes to draw an appropriate color.